packages/mux-player/src/template.ts

```diff
@@ -71,24 +71,8 @@ export const content = (props: MuxTemplateProps) => html`
       player-software-version="${props.playerSoftwareVersion ?? false}"
       env-key="${props.envKey ?? false}"
       custom-domain="${props.customDomain ?? false}"
-      src="${!!props.src
-        ? props.src
-        : props.playbackId
-        ? getSrcFromPlaybackId(props.playbackId, {
-            maxResolution: props.maxResolution,
-            domain: props.customDomain,
-            token: props.tokens.playback,
-          })
-        : false}"
-      cast-src="${!!props.src
-        ? props.src
-        : props.playbackId
-        ? getSrcFromPlaybackId(props.playbackId, {
-            maxResolution: props.maxResolution,
-            domain: props.customDomain,
-            token: props.tokens.playback,
-          })
-        : false}"
+      src="${!!props.src ? props.src : props.playbackId ? toMuxVideoURL(props) : false}"
```

Comment:
NOTE: with both the refactored function signature and removing the similarly-purposed `getSrcFromPlaybackId`, this should make code maintenance easier.